### PR TITLE
[spiflash] Re-align frame size with OpenTitan, adjust wait times

### PR DESCRIPTION
**Pull request by vogelpi**
This PR is required to make ot-sca work again with current OpenTitan master. We should probably merge #37 first. WDYT @alphan ?

### CODE REVIEW

File: cw/cw305/util/spiflash.py [hunk 1]
```diff
@@ -126,8 +126,8 @@ class _Bootstrap:
   _PIN_BOOTSTRAP='USB_A20'
   # Delays below are in seconds.
   _BOOTSTRAP_DELAY=0.1
-  _FIRST_FRAME_DELAY=0.1
-  _INTER_FRAME_DELAY=0.01
+  _FIRST_FRAME_DELAY=0.2
+  _INTER_FRAME_DELAY=0.02
```
Comment (alphan) [inline cw/cw305/util/spiflash.py:130]: just curious: did you adjust these just to be on the safe side (no need to revert) or were these tight already and stopped working after the frame size update?

**Review by alphan — APPROVED** [review]
LGTM, thanks @vogelpi.

File: cw/cw305/util/spiflash.py (hunk 1 above)
Comment (vogelpi) [reply cw/cw305/util/spiflash.py:130]: These delays were indeed too small for the increased frame size. But I didn't experiment to reduce them to the minimum.

**Comment by vogelpi** [comment]
Thanks for the quick review @alphan , I am merging this now.

**Comment by vogelpi** [comment]
Ah wait, no. The bigger frame size would probably break the current binaries. Let's wait until we have new binaries first.

**Comment by alphan** [comment]
> Ah wait, no. The bigger frame size would probably break the current binaries. Let's wait until we have new binaries first.

Good point, it is nice to have commits that work and should be easy to achieve in this case.

**Comment by vogelpi** [comment]
I am closing this PR as the relevant commit has been merged into #43. It's better to do one atomic change (re-align bitstream, binary and spiflash config).

**Pull request closed without merging**